UI/DebugOverlay.cpp

```diff
@@ -437,7 +437,7 @@ void DrawFPS(UIContext *ctx, const Bounds &bounds) {
 
 	char fpsbuf[256]{};
 	if (g_Config.iShowStatusFlags == ((int)ShowStatusFlags::FPS_COUNTER | (int)ShowStatusFlags::SPEED_COUNTER)) {
-		snprintf(fpsbuf, sizeof(fpsbuf), "%0.0f/%0.0f (%0.1f%%)", actual_fps, fps, vps / (59.94f / 100.0f));
+		snprintf(fpsbuf, sizeof(fpsbuf), "%0.0f/%0.0f (%0.1f%%)", actual_fps, fps, vps / (g_Config.iDisplayRefreshRate / 100.0f));
```

Comment:
This was not 60, might need to scale it, not sure tbh :)

Reply:
Right, might be related to this issue https://github.com/hrydgard/ppsspp/issues/15081

Reply:
Yup, I had tried subtracting 0.94, but higher framerates like 165 would indicate 100.5%. As is, 60 FPS doesn't always show 100%.

I could try scaling it or subtracting 0.94 anyway if you wish.

Reply:
Sorry, that was quite the brainfart.

Reply:
Mhhh, any intuition on why a subtraction is correct here? It feels like a scaling factor thing to me:

`g_Config.iDisplayRefreshRate/60.f*59.94f`

Maybe the subtraction is correct, I'm not really and expert in this thing :)

Reply:
I don't know why we have to subtract a tiny bit to settle to 100%, but it scales to any refresh rate just fine thankfully !